tempesta_fw/sock_clnt.c

```diff
@@ -41,11 +41,9 @@
  * TfwConnection extension for client sockets.
  *
  * @conn	- The base structure. Must be the first member.
- * @ka_timer	- The keep-alive timer for the connection.
  */
 typedef struct {
 	TfwConnection		conn;
-	struct timer_list	ka_timer;
 } TfwCliConnection;
```

Comment:
So `TfwCliConnection` becomes single-member structure and there is no sense to pollute name space. It's better to remove the structure.